### ISSUE TITLE
#CHECK behaviour

### ISSUE THREAD

**Issue by GuillaumeGen** [Deducteam/Dedukti#27]
I don't understand the behaviour of `#CHECK`.
If `a` is of type `A` then `#CHECK a, A` print `YES` but if `a` is not of type `A` then `#CHECK a, A` raise an error:
```
Error while typing a.
Expected: A
Inferred: ...
```

I would prefer `#CHECK` to simply answer `NO` rather than raising an exception.

**Comment by rafoo** [comment]
According to the manual:
```
#CHECK t1, t2.  (;  display "OK" if t1 has type t2, "KO" otherwise. ;)
```

So I guess you spotted two bugs at once:
- the manual is wrong, the answers should be 'YES' and 'NO', not 'OK' and 'KO',
- the implementation is wrong, 'NO' should be answered instead of failure.

**Comment by GuillaumeGen** [comment]
I created PR #28 to fix this problem.
It creates a command `#ASSERT` which fails and the command `#CHECK` simply answer `YES` or `NO`

**Comment by rlepigre** [comment]
This has been fixed now since #28 is merged (through #38).
